extension-base/src/main/java/com/azure/autorest/extension/base/plugin/NewPlugin.java

```diff
@@ -80,6 +80,10 @@ public boolean getBooleanValue(String key, boolean defaultValue) {
         }
     }
 
+    public List<String> getListValue(String key) {
```

Comment:
Might not worth adding to `NewPlugin` if it only works for `String`. If you want to put it here, maybe make it more generic by using `List<T>`? Or just have the method as private in `JavaSettings`.

Reply:
Change to inline code now